core/docker/data/conf/configuration.yml

```diff
@@ -1,12 +1,29 @@
 server:
   adminMinThreads: 1
   adminMaxThreads: 64
+<#if env.HTTPS?has_content && env.HTTPS == "true">
```

Comment:
problem here is we're fighting the dropwizard automatic wireup from the config.  other projects do this kind of thing by supplying optional blocks via independent json files, then when they configure it all they stitch the files into one logical blob and work on that.  thats how sensu works and i really like that model.

what we can consider doing (since our configs are becoming crazy parameterized) is look into a different config runner with dropwizard, and maybe take some extra onus of setup on ourselves.